### PR TITLE
Include remove_locks.py in the Scripts to be Installed and Fix Date/Time String

### PR DESCRIPTION
**Pull request by Dala0**
Includes remove_locks.py in the scripts to be installed. Also fixes the date/time string to be added to the database by adding a 'T' in the right place. If the 'T' is not present then other parts of the software break.

@fredex42

### CODE REVIEW

**Comment by Dala0** [comment]
[Asset Sweeper Missing remove_locks.py Script](https://trello.com/c/4Y2NTfwe/3009-asset-sweeper-missing-removelockspy-script)

**Comment by Dala0** [comment]
[Asset Sweeper remove_locks.py Script Adds Broken Date/Time String](https://trello.com/c/9fCET9Zm/3010-asset-sweeper-removelockspy-script-adds-broken-date-time-string)

File: src/remove_locks.py [hunk 1]
```diff
@@ -32,7 +32,10 @@
 if cursor.rowcount==0:
     print "Script is locked. Removing lock."
 
-    cursor.execute("insert into system (key,value,pid) values ('run_end',%s,%s)", (datetime.datetime.now(),str(pid),)
+    date_time_string = "%s" % (datetime.datetime.now())
+    date_time_string_fixed = date_time_string[:10] + 'T' + date_time_string[11:]
```
Comment (fredex42) [inline src/remove_locks.py]: `datetime.isoformat()` would be a better option for this - https://docs.python.org/2/library/datetime.html#datetime.datetime.isoformat

File: src/setup.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
       scripts=['asset_folder_sweeper.py','asset_folder_verify_files.py',
                'asset_folder_vsingester.py','asset_permissions.pl',
                'prelude_importer.py','premiere_get_referenced_media.py',
-               'fix_unattached_media.py','vs_resync_deleted.py', 'find_flushable_unimported_media.py'],
+               'fix_unattached_media.py','vs_resync_deleted.py', 'find_flushable_unimported_media.py', 'remove_locks.py'],
```
Comment (fredex42) [inline src/setup.py]: 👍

File: src/remove_locks.py (hunk 1 above)
Comment (Dala0) [reply src/remove_locks.py]: Done.

**Pull request merged**